lib/system.g

```diff
@@ -66,14 +66,14 @@ BIND_GLOBAL( "GAPInfo", rec(
       rec( short:= "y", long := "lines", default := "", arg := "<num>", help := ["set number of lines"] ),
       ,
       rec( short:= "g", long := "gasinfo", default := 0,
-           help := ["show GASMAN messages (full/all/no garbage","collections)"] ),
+           help := ["show GASMAN messages (full/all/no garbage","collections)", "(available only if GASMAN is running)"] ),
```

Comment:
I think this could be misunderstood, as in "perhaps GASMAN is some software that I need to install and run on my computer for this to work? but where do I get it). Perhaps this is slightly clearer in this regard:
```suggestion
           help := ["show GASMAN messages (full/all/no garbage","collections)", "(only available if GAP uses GASMAN)"] ),
```

Another alternative would be to not show this text if GASMAN is used; and otherwise show a text more like "not available in this GAP build"; or perhaps "not available with the active GAP memory manager" and then at some point print the memory manager as part of the output of this usage text (i.e. at the start or end, say something like "GAP was compiled using the FOOBAR memory manager")

Reply:
I have changed the formulation as proposed.
Leaving out the not supported options from the help list may be irritating for users --suppose I have read somewhere how to start GAP, including some fancy command line options, thenI get an error message, look at the help screen, and the relevant options are not listed at all.